### PR TITLE
remove cron

### PR DESCRIPTION
**Pull request by briskt**
### Changed
- Changed the container to run `/data/yii batch/full` instead of the cron service. This requires a change to the deployment of this service. Rather than using a constantly-running ECS service, it should be triggered to run on a schedule by Cloudwatch.

### Removed
- Removed the cron service and all associated configuration.

---

### Feature PR Checklist
- [x] Documentation (README, etc.)
- [x] Unit tests created or updated
- [x] Run `make composershow`

### CODE REVIEW

File: application/run-tests.sh [hunk 1]
```diff
@@ -1,15 +1,13 @@
 #!/usr/bin/env bash
 
+set -e
```
Comment (mtompset) [inline application/run-tests.sh:3]: Perhaps a comment refreshing people's memory about the functionality of the set -e and set -x commands would be helpful. Lest they want to put the rc=$? code back in.

**Review by mtompset — APPROVED** [review]
Thanks for the clarifying comments being added.

**Review by jason-jackson — APPROVED** [review]
A few log messages are no longer printed, would it be beneficial to add them to make file or somewhere else?
- On start
- On error
- On completion

**Review by forevermatt — APPROVED** [review]
Given that this will be a breaking change, I think this is fine. It should also probably conserve some our cluster resources, too. 👍

**Comment by briskt** [comment]
> A few log messages are no longer printed, would it be beneficial to add them to make file or somewhere else?
> 
> * On start
> * On error
> * On completion

Are you referring to the `echo` lines in the deleted script files? I think the PHP code does a fine job of logging as it is. Is that what you meant?

**Comment by jason-jackson** [comment]
> > A few log messages are no longer printed, would it be beneficial to add them to make file or somewhere else?
> > 
> > * On start
> > * On error
> > * On completion
> 
> Are you referring to the `echo` lines in the deleted script files? I think the PHP code does a fine job of logging as it is. Is that what you meant?

Yes, that was correct, I was thinking it would be fine too with just the PHP logs, but thought I would mention it.

**Pull request merged**